test_runner/src/main/kotlin/ftl/shard/Shard.kt

```diff
@@ -35,6 +36,20 @@ class com.foo.ClassName#testMethodToSkip
 
 */
 
+fun createShardsByTestForShards(
+    args: AndroidArgs
+): List<TestShard> {
+    val testShards = mutableListOf<TestShard>()
+    args.testTargetsForShard.forEach { list ->
+        val testMethods = mutableListOf<TestMethod>()
+        list.forEach { name ->
+            testMethods.add(TestMethod(name, 0.0, false))
+        }
+        testShards.add(TestShard(testMethods[testMethods.lastIndex].time, testMethods))
+    }
+    return testShards
+}
```

Comment:
```suggestion
fun createShardsByTestForShards(
    args: AndroidArgs
): List<TestShard> =
    args.testTargetsForShard.map { list ->

        val testMethods = list
            .map { name -> TestMethod(name, 0.0, false) }
            .toMutableList()

        TestShard(
            time = testMethods[testMethods.lastIndex].time,
            testMethods = testMethods
        )
    }
```

Pls use expressions and avoid operations on mutable collections where possible.